packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -144,6 +146,12 @@ export default class TooltipAnchor extends React.Component<Props, State> {
         // If we changed state, call our subscriber and let them know.
         if (oldState !== newState) {
             this.setState({active: newState});
+
+            if (newState) {
```

Comment:
I wonder if we should use `newActive` and `oldActive`?

Reply:
This is changing in an upcoming PR (where we add all the delays and things), so I'd wait until after that.